docs/en/index.md

```diff
@@ -6,109 +6,109 @@
 
-# Experiments
+Whether you're just getting started or already knee-deep in data analysis, the Advanced Analytics Workspace has everything you need to take your work to the next level. From powerful tools for data pipelines to cloud storage for your datasets, our platform has it all. Need to collaborate with colleagues or publish your results? No problem. We offer seamless collaboration features that make it easy to work together and share your work with others.
 
-## Process data using `R`, `Python`, or `Julia`
+No matter what stage of your data science journey you're at, the Advanced Analytics Workspace has the resources you need to succeed.
 
-[![R, Python, or Julia in Jupyter notebooks](images/Jupyter.PNG)](1-Experiments/Jupyter.md)
-Once you have Kubeflow set up, use
-**[Jupyter Notebooks](1-Experiments/Jupyter.md)** to create and share documents
-that contain live code, equations, or visualizations.
-![Jupyter Notebooks](images/jupyter_in_action.png)
+## Getting Started with the AAW
 
-## Process data using 'R' or 'Python'
+<center>
+![image](https://user-images.githubusercontent.com/8212170/158243976-0ee25082-f3dc-4724-b8c3-1430c7f2a461.png)
+</center>
+
+### The AAW Portal
+
+The AAW portal homepage is available for internal users only. However, external users with a cloud account granted access by the business sponsor can access the platform through the analytics-platform URL.
 
-[![R or Python in R Studio](images/RStudio.PNG)](1-Experiments/RStudio.md)
-**[R Studio](1-Experiments/RStudio.md)** gives you an integrated development
-environment for R and Python. Use the r-studio-cpu image to get an R Studio
-environment.
+!!! info annotate "AAW Portal Homepage"
+    - [**Portal Homepage for Statistics Canada Employees**](https://www.statcan.gc.ca/data-analytics-service/aaw)
+    - [**Portal Homepage for External Users**](https://analytics-platform.statcan.gc.ca/covid19)
 
-## Run a virtual desktop
+!!! info "Kubeflow Dashboard"
+    - [**Kubeflow Dashboard**](https://kubeflow.aaw.cloud.statcan.ca/) Use this link once you have your cloud account!
 
-[![Virtual Desktop](images/VirtualDesktop.PNG)](1-Experiments/Remote-Desktop.md)
-You can run a full Ubuntu desktop, with typical applications, right inside your
-browser, using [**ML Workspaces**](1-Experiments/Remote-Desktop.md)
+Getting started with the Advanced Analytics Workspace (AAW) is easy and quick. First, you'll want to set up Kubeflow for MLOps and Jupyter Notebooks. Kubeflow makes it easy to deploy and manage end-to-end machine learning workflows, while Jupyter Notebooks provide a flexible and powerful environment for data analysis and experimentation. Once you have Kubeflow and Jupyter Notebooks set up, you can start exploring the many resources and tools available through the AAW portal. Additionally, we encourage you to join our Slack channel to connect with other data scientists and analysts, ask questions, and share your experiences with the AAW platform.
```

Comment:
`First, you'll want to set up Kubeflow for MLOps and Jupyter Notebooks.` no real setup, just log in. Also no MLOps since on pipelines.